### ISSUE TITLE
MAKE NEW CORPORA API

### ISSUE THREAD

**Issue by maria-terekhina** [lingcorpora/lingcorpora.py#4]
* look at list, choose a corpus, make an API
* till 26.02.2018

**Comment by ustya-k** [comment]
1. Several **Russian** corpora http://corpus.leeds.ac.uk/ruscorpora.html
2. Several **English** corpora http://corpus.leeds.ac.uk/protected/query.html
3. Internet corpora for several languages http://corpus.leeds.ac.uk/internet.html
4. ~~**Eng-Chinese** corpus http://www.jukuu.com/~~
5. **German** corpus COSMAS II https://cosmas2.ids-mannheim.de/cosmas2-web/ (structure http://www.ids-mannheim.de/cosmas2/projekt/referenz/dokutexte.html)
6. **German** corpus c4 https://c4.chtk.ch/en/ (structure https://www.korpus-c4.org/index.php/en/structure)
7. **Armenian** http://eanc.net/
8. Caucasus corpora (Aghul, Dargwa, Archi, Bagvalal, Godoberi, Sanzhi Dargwa, Tsakhur, Chirag) http://web-corpora.net/caucasus_corpora.html
9. **Avar** http://web-corpora.net/AvarCorpus/search/
10. **Adyghe** http://web-corpora.net/AdygheCorpus/search/?interface_language=ru 
11. 10+ other Arkhangelsky’s corpora http://web-corpora.net/
12. **Macedonian** http://makedonski.info/
13. **Czech** https://kontext.korpus.cz/
14. **Slovenian** http://nl.ijs.si/noske/all.cgi/first_form
15. **Persian** http://pars.ie/lr/corpora/run.cgi/first_form
16. **Hungarian** http://hnc.nytud.hu/index_eng.html (registration) http://clara.nytud.hu/mtsz/run.cgi/first_form
17. 15+ other NoSketch engine corpora http://aranea.juls.savba.sk/guest/index.html (registration)
18. **Hittite** http://hittitecorpus.ru/ (registration)
19. **Ket** and **Evenki** http://corket.ru/
20. **Selkup** http://selkup.org/corpus-search
21. [A Glossed Audio Corpus of Ainu Folklore]( http://ainucorpus.ninjal.ac.jp/en/)
22. [Мандешные корпуса обновились и пополнились](http://maslinsky.spb.ru/cormand/)

1 — 3 Leeds corpora http://corpus.leeds.ac.uk/list.html
7 — 11 Arkhangelsky’s corpora
13 — 17 NoSketch engine corpora (Bamana and Maninka are already created) https://nlp.fi.muni.cz/trac/noske

**Comment by agricolamz** [comment]
@ustya-k, @Pandaklez , @Tuylenen, 

have you choose a corpus? Have you start working on creating API? ASAP!

till 26.02.2018

**Comment by Pandaklez** [comment]
Yes, I've recently started to work on spreadthesign. But I think I'm a
little bit stuck. Maybe I will write you personal letter with a question
soon.

21 февр. 2018 г. 7:46 пользователь "George Moroz" <notifications@github.com>
написал:

**Comment by agricolamz** [comment]
@ustya-k, @Tuylenen, pleases answer!

**Comment by ustya-k** [comment]
Sorry, I thought I already answered. I took Tatar corpus, will post it tomorrow evening. After that I am going to add other similar Arkhangelsky’s corpora.

**Comment by Tuylenen** [comment]
I'm trying to work on the Czech National Corpus, will probably post the result in a couple of days.

**Comment by agricolamz** [comment]
There is some javascript there. So good luck.
